packages/uui-color-swatch/lib/uui-color-swatch.element.ts

```diff
@@ -282,14 +251,11 @@ export class UUIColorSwatchElement extends LabelMixin(
         line-height: 0;
         transition: fill 120ms, opacity 120ms;
         fill: #fff;
+        filter: invert(1) grayscale(1) contrast(9);
```

Comment:
It would seem the filter doesn't quite work for dark colors:

![image (6)](https://github.com/umbraco/Umbraco.UI/assets/752371/3eb485f2-6282-4585-a585-3990a4940ca4)

Reply:
Versus as it was before:
![image (7)](https://github.com/umbraco/Umbraco.UI/assets/752371/7f25a719-22af-43fc-8b89-0109d92b4000)